typescript/src/lib/bitcoin/ecdsa-key.ts

```diff
@@ -9,7 +9,13 @@ import { BitcoinNetwork, toBitcoinJsLibNetwork } from "./network"
  * @param publicKey - Public key that should be checked.
  * @returns True if the key is a compressed Bitcoin public key, false otherwise.
  */
-function isCompressedPublicKey(publicKey: string): boolean {
+function isCompressedPublicKey(publicKey: string | Hex): boolean {
```

Comment:
Do we need to support `publicKey` as a `string`? Can we accept just `Hex`? Same for other functions in this file.

Reply:
We can just use `Hex`.
Removed `string` argument in 527cdd99f9102ddcef800d91696a720c6602a092 for all the functions in this file.